bin/templates/project/__CLI__.xcodeproj/project.pbxproj

```diff
@@ -374,7 +374,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "cordova/lib/copy-www-build-step.sh";
+			shellScript = "node cordova/lib/copy-www-build-step.js";
```

Comment:
Hello @shazron, @nikhilkh,
Thanks for your review.
Regarding node' path - I propose to leave it as is taking into account that stand-alone case is uncommon.
We can address it later in case of issues.